keras/applications/mobilenet.py

```diff
@@ -0,0 +1,275 @@
+        `image_data_format='channels_last'` in your Keras config
+        at ~/.keras/keras.json.
+        # Arguments
+            input_shape: optional shape tuple, only to be specified
+                if `include_top` is False (otherwise the input shape
+                has to be `(224, 224, 3)` (with `channels_last` data format)
+                or (3, 224, 224) (with `channels_first` data format).
+                It should have exactly 3 inputs channels,
+                and width and height should be no smaller than 32.
+                E.g. `(200, 200, 3)` would be one valid value.
+            alpha: width multiplier of the MobileNet.
+            depth_multiplier: depth multiplier for depthwise convolution
+                (also called the resolution multiplier)
+            dropout: dropout rate
+            include_top: whether to include the fully-connected
+                layer at the top of the network.
+            weights: `None` (random initialization) or
+                `imagenet` (ImageNet weights)
+            input_tensor: optional Keras tensor (i.e. output of `layers.Input()`)
+                to use as image input for the model.
+            pooling: Optional pooling mode for feature extraction
+                when `include_top` is `False`.
+                - `None` means that the output of the model will be
+                    the 4D tensor output of the
+                    last convolutional layer.
+                - `avg` means that global average pooling
+                    will be applied to the output of the
+                    last convolutional layer, and thus
+                    the output of the model will be a 2D tensor.
+                - `max` means that global max pooling will
+                    be applied.
+            classes: optional number of classes to classify images
+                into, only to be specified if `include_top` is True, and
+                if no `weights` argument is specified.
+        # Returns
+            A Keras model instance.
+        '''
+
+    if K.backend() != 'tensorflow':
+        raise AttributeError('Only Tensorflow backend is currently supported, '
+                             'as other backends do not support depthwise convolution.')
+
+    if weights not in {'imagenet', None}:
+        raise ValueError('The `weights` argument should be either '
+                         '`None` (random initialization) or `imagenet` '
+                         '(pre-training on ImageNet).')
+
+    if weights == 'imagenet' and include_top and classes != 1000:
+        raise ValueError('If using `weights` as ImageNet with `include_top`'
+                         ' as true, `classes` should be 1001')
+
+    if weights == 'imagenet':
+        assert depth_multiplier == 1, "If imagenet weights are being loaded, depth multiplier must be 1"
+
+        assert alpha in [0.25, 0.50, 0.75, 1.0], "If imagenet weights are being loaded, alpha can be one of" \
```

Comment:
Line too long
Don't use `\` for breaking lines
Use a ValueError instead of an assert